test/e2e/specs/site-editor/hybrid-theme.spec.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Hybrid theme', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'gutenberg-test-themes/emptyhybrid' );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test( 'should not show the Add Template Part button', async ( {
+		admin,
+		page,
+	} ) => {
+		await admin.visitAdminPage(
+			'/site-editor.php',
+			'postType=wp_template_part&path=/wp_template_part/all'
+		);
```

Comment:
The `visitSiteEditor` waits for the canvas area to load, but the all template parts menu does not. Therefore, I used `visitAdminPage` instead.

Reply:
This is correct usage 👍